apiserver/common/tools.go

```diff
@@ -239,14 +236,12 @@ func NewToolsFinder(
 
 // FindTools returns a List containing all tools matching the given parameters.
 func (f *ToolsFinder) FindTools(args params.FindToolsParams) (params.FindToolsResult, error) {
-	result := params.FindToolsResult{}
 	list, err := f.findTools(args)
 	if err != nil {
-		result.Error = apiservererrors.ServerError(err)
-	} else {
-		result.List = list
+		return params.FindToolsResult{Error: apiservererrors.ServerError(err)}, nil
 	}
-	return result, nil
+
+	return params.FindToolsResult{List: list}, nil
```

Comment:
^- there is no functional change here, correct?

Reply:
Nope.